### PR TITLE
serialize camel case style property

### PR DESCRIPTION
**Pull request by Javey**
As is mentioned in https://github.com/Matt-Esch/virtual-dom/issues/359

### CODE REVIEW

**Comment by substack** [comment]
I also started on a patch like this, but it has caveats: https://github.com/Matt-Esch/virtual-dom/issues/359#issuecomment-169721411

**Comment by Javey** [comment]
@substack 
Without `Proxy` min-document can't really support camel-cased styles. But I think it is unnecessary to make it so complicated. This patch can do correct things, if you write styles in one casing.

**Comment by substack** [comment]
That is true. I think this patch is better than the current behavior and should be merged, but there are also caveats!

**Comment by Javey** [comment]
I updated this patch for unescaped html.

File: serialize.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ function serializeElement(elem) {
             strings.push.apply(strings, elem.childNodes.map(serializeNode))
         } else if (elem.textContent || elem.innerText) {
             strings.push(escapeText(elem.textContent || elem.innerText))
+        } else if (elem.innerHTML) {
+            strings.push(elem.innerHTML)
```
Comment (Raynos) [inline serialize.js]: Why this change ?

**Comment by Raynos** [comment]
lgtm.

**Comment by Raynos** [comment]
Published 2.18

File: serialize.js (hunk 1 above)
Comment (Javey) [inline serialize.js]: For html injection, as is mentioned in https://github.com/Matt-Esch/virtual-dom/blob/master/docs/vtext.md#html-injection
